i18n/OWNERS

```diff
@@ -0,0 +1,13 @@
+# This is the directory for internationalized site strings.
+
+reviewers:
+- sig-docs-pr-reviews
```

Comment:
Should we rename this group to `sig-docs-en-reviews` (as its related appovers group is called `sig-docs-en-owners`)? It's OK as-is, but it may cause kinda confusing in the future.

Reply:
We need to keep an easily findable group for docs PRs, but I agree that because it doesn't follow convention that it's unnecessarily confusing. I'll add a separate team for EN reviews following the same naming conventions.